repos/rust-lang/specification.toml

```
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "spec"
+description = "The Rust specification"
+bots = ["rustbot", "rfcbot"]
+
+[access.teams]
+spec = 'maintain'
```

May I suggest adding branch protection for the `main` branch.
```suggestion
spec = 'maintain'

[[branch-protections]]
pattern = 'main'
```

---

I think we'll just enable that at a later point. While we're setting everything up, it might be easier to just be able to push directly. ^^